pkg/transcriptionsystems/bipa/vowels.tsv

```diff
@@ -157,6 +157,7 @@ ỹː	rounded	close	front		nasalization:nasalized,duration:long
 ɘ̃ɘ̃	unrounded	close-mid	central	+	nasalization:nasalized	
 ɘ̃ː	unrounded	close-mid	central		nasalization:nasalized,duration:long	
 ə	unrounded	mid	central			
+ә	unrounded	mid	central	+		
```

Comment:
This should not be handled as an alias, it shoudl be handled as a normalization aspect. We have the file `normalize.tsv` for this purpose.

Reply:
I will move it.